app/jobs/resolution_proofing_job.rb

```diff
@@ -190,6 +199,8 @@ def resolution_proofer
     @resolution_proofer ||=
       if IdentityConfig.store.proofer_mock_fallback
         Proofing::Mock::ResolutionMockClient.new
+      elsif IdentityConfig.store.replace_lexisnexis_rdp_instant_verify_with_ddp
+        lexisnexis_ddp_proofer
```

Comment:
this is a little speculative...maybe outside the scope of this ticket? Do we want to worry about integration all at once here or maybe should we break this into multiple PRs?

Reply:
ha you are right.  making my life more difficult here.  perhaps commenting it out?